lnrpc/lightning.proto

```diff
@@ -1545,6 +1545,13 @@ message Channel {
 
     // This is the peer SCID alias.
     uint64 peer_scid_alias = 35 [jstype = JS_STRING];
+
+    /*
+    An optional note-to-self to go along with the channel containing some
+    useful information. This is only ever stored locally and in no way impacts
+    the channel's operation.
+    */
+    string memo = 36;
```

Comment:
I think this commit message suggests that the commit does a lot more than what it actually does. Consider revising the commit message.

Reply:
Good catch, updated!